### PR TITLE
Fix #205: Add jest -watch option

### LINKED ISSUE #205: Add watch option to Jest
Watch will watch for the files that has changes and rerun tests related to that file

### PR DESCRIPTION
**Pull request by lozinska**
Adding jest --watch option to the code will allowed user to do detailed test on the files.
After using command `npm run watch` user will see pop up menu that will give options  what commands can be run in order to perform analysis on the files:
r04

### CODE REVIEW

**Comment by lozinska** [comment]
@humphd interesting thing. Thank you for letting me know.  I haven't even notice that

**Comment by lozinska** [comment]
@humphd I by mistake add wrong file, so it didn't update package.json

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "postlint": "npm run stylelint",
     "prettier": "prettier --write \"./**/*.{md,json,html,css,js,yml}\"",
     "pretest": "npm run lint",
+    "watch": "jest --watch",
```
Comment (humphd) [inline package.json]: `cross-env MOCK_REDIS=1 jest --watch` so that it mocks redis

**Comment by humphd** [comment]
What's the hold up with this?  Can we please get this finished and merged?

**Comment by lozinska** [comment]
@humphd yes sure, I'll do it shortly today

File: package.json (hunk 1 above)
Comment (humphd) [reply package.json]: Remove this line, since you've added `jest-watch` below (we don't need both).

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ Some helpful guides:
 1. Run `npm install`.
 1. Run `npm test`
 1. IF eslint detect some issues run `npm run eslint-fix` before manually fixing the issue (Will save you time :smile:) and then run `npm test` again.
+1. Run `npm run watch` to watch files for any changes and rerun tests related to changed files.
```
Comment (humphd) [inline docs/CONTRIBUTING.md]: Update this to say `npm run jest-watch`

**Pull request merged**